python_modules/dagster-airflow/dagster_airflow/dagster_plugin.py

```diff
@@ -204,42 +203,45 @@ def execute(self, context):
                 if 'status' in output:
                     self.log.info("%s", output['status'])
 
-        with TemporaryDirectory(prefix='airflowtmp', dir=self.host_tmp_dir) as host_tmp_dir:
-            self.environment['AIRFLOW_TMP_DIR'] = self.tmp_dir
-            self.volumes.append('{0}:{1}'.format(host_tmp_dir, self.tmp_dir))
+        # FIXME: need to figure out what to do here. We probably want to provide some knobs to
```

Comment:
Can we structure this code so that there is less thrash in the ModifiedDockerOperator? We could have a wrapper context manager in the base class for example.

```
@contextmanager
def get_host_tmp_dir(self):
   with TemporaryDirectory(prefix='airflowtmp', dir=self.host_tmp_dir) as h:
      yield h
```

and then override it in the derived class:
```
@contextmanager
   def get_host_tmp_dir(self):
      yield self.host_tmp_dir
```
or whatever is appropriate